internal/event/events.go

```diff
@@ -0,0 +1,6 @@
+package event
+
+type LpaUpdated struct {
+	Uid  string
```

Comment:
should these have `json:` tags to be "uid" and "changeType"?

Reply:
and a small nit on this (that people don't always stick to) is that initialisms are named in a consistent case, so `UID` here https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
Yes you're right, thanks
> should these have `json:` tags to be "uid" and "changeType"?

About the initialisms it might be better to let it be consistent with other instances where we use `Uid` especially since it's like that in other repos, although I'd also personally prefer to capitalise it